rubix-spi/src/main/java/com/qubole/rubix/spi/CacheConfig.java

```diff
@@ -87,6 +87,10 @@
   private static final String KEY_MAX_CACHE_SIZE = "rubix.cache.max.size";
   private static final String KEY_CACHE_FILE_SPLIT_SIZE = "rubix.cache.filesplit.size";
   private static final String KEY_CLUSTER_NODE_REFRESH_TIME = "rubix.cluster.node.refresh.time";
+  private static final String KEY_WORKER_NODEINFO_EXPIRY_PERIOD = "rubix.worker.nodeinfo.expiry.period";
+  private static final String KEY_RUBIX_CLUSTER_TYPE = "rubix.cluster.type";
+  private static final String KEY_RUBIX_WORKER_NODELIST_FETCH_MAX_RETRIES = "rubix.worker.nodelist.fetch.max-retries";
+  private static final String KEY_RUBIX_WORKER_NODELIST_FETCH_WAIT_INTERVAL = "rubix.worker.nodelist.fetch.wait.interval";
```

Comment:
Add documentation for configuration options where appropriate (especially for `rubix.cluster.type`, since this option takes an integer instead of a meaningful name)